gr-qtgui/grc/qtgui_push_button.block.yml

```diff
@@ -2,6 +2,9 @@ id: variable_qtgui_push_button
 label: QT GUI Push Button
 flags: [ show_id, python ]
 
+deprecated: True
+deprecation_note: "Use the QT GUI Msg Push Button block instead."
+
```

Comment:
That's not the syntax with which you can deprecate a block! Look at the syntax for other deprecated blocks, e.g., in gr-digital/grc/digital_crc16_async_bb.block.yml